src/cart/index.jsx

```diff
@@ -1,11 +1,139 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import { numberWithCommas } from "../functions/index";
+
+const buildOptions = () => {
+  var arr = [];
+  for (let i = 1; i <= 90; i++) {
+    arr.push(
+      <option key={i} value={i}>
+        {i}
+      </option>
+    );
+  }
+  return arr;
+};
+
+const CartItemCount = ({ value, setQuantity, id }) => {
```

Comment:
Same here this function belongs to `CartItem` so please put them in `CartItem.jsx`.